birdhouse/pavics-compose.sh

```diff
@@ -87,6 +88,7 @@ OPTIONAL_VARS='
   $AUTODEPLOY_EXTRA_SCHEDULER_JOBS
   $PROXY_READ_TIMEOUT_VALUE
   $PROXY_ROOT_LOCATION
+  $SECURE_DATA_PROXY_AUTH_INCLUDE
```

Comment:
This should not be here since it is dynamically added by `optional-components/secure-data-proxy/`

Reply:
I see why, because you had to modify `proxy/conf.d/all-services.include.template`.

Reply:
It must be specified even when not used by `optional-components/secure-data-proxy`, otherwise it is not substituted in https://github.com/bird-house/birdhouse-deploy/blob/secure-data-proxy/birdhouse/config/proxy/conf.d/all-services.include.template?rgh-link-date=2023-02-10T04%3A08%3A10Z#L42, which in turn causes nginx to complain about invalid syntax.